src/Hintman/HLint.hs

```diff
@@ -29,6 +31,7 @@ For this we need:
 4. Run @Hlint@ tool on the content.
 5. Return the list of all 'Idea's.
 
+TODO: Use "--ignore=Parse error" somewhere.
```

Comment:
Yes, this would be nice! Can we convert these TODOs to issues? I think it will be useful to have them all implemented :slightly_smiling_face: